### PR TITLE
feat: Add readme for tracer

### PR DESCRIPTION
**Pull request by spirosoik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,57 @@
...
+[Go Modules](https://github.com/golang/go/wiki/Modules), you may specify a
+version number as well:
+
+```console
+$ go get github.com/lumigo-io/go-tracer-beta@master
+```
+
+## Usage
+
+You need a lumigo token which you can find under the `Project Settings` and `Tracing` tab in lumigo portal. Then you need just to wrap your Lambda:
```
Comment (nirLumigo) [inline README.md:28]:
```suggestion
You need a lumigo token which you can find under the `Project Settings` and `Tracing` tab in lumigo platform. Then you need just to wrap your Lambda:
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,57 @@
...
+
+```go
+type MyEvent struct {
```
Comment (nirLumigo) [inline README.md:31]: Maybe we should also add the import line?
```
lumigotracer "github.com/lumigo-io/go-tracer-beta"
```

File: README.md [hunk 3]
```diff
@@ -1,12 +1,57 @@
...
+}
+
+func main() {
+	wrappedHandler := lumigotracer.WrapHandler(HandleRequest, &lumigotracer.Config{
+		Token:       "<your-token>",
+	})
+	lambda.Start(wrappedHandler)
+}
+```
+
```
Comment (nirLumigo) [inline README.md:46]: They also need to add the environment variable: `LUMIGO_USE_TRACER_EXTENSION: true`.
And the layer: `arn:aws:lambda:us-east-1:114300393969:layer:lumigo-tracer-extension:36`.
In a serverless.yml file it looks like this:
```
    environment:
      LUMIGO_USE_TRACER_EXTENSION: true
    layers:
      - arn:aws:lambda:us-east-1:114300393969:layer:lumigo-tracer-extension:36
```

**Review by nirLumigo — CHANGES_REQUESTED** [review]
Nice! :)

File: README.md (hunk 3 above)
Comment (spirosoik) [reply README.md:46]: btw we should add the same layers table as in python tracer actually. Like this https://github.com/lumigo-io/python_tracer#with-lambda-layers

**Comment by spirosoik** [comment]
@nirLumigo please have a look

**Review by nirLumigo — APPROVED** [review]

**Pull request merged**